C4.md

```diff
@@ -0,0 +1,131 @@
+
+### 2.2. Licensing and Ownership
+
+1. The project SHALL use a share-alike license such as the MPLv2, or a GPLv3 variant thereof (GPL, LGPL, AGPL).
+2. All contributions to the project source code (“patches”) SHALL use the same license as the project.
+3. All patches are owned by their authors. There SHALL NOT be any copyright assignment process.
+4. Each Contributor SHALL be responsible for identifying themselves in the project Contributor list.
+
+### 2.3. Patch Requirements
+
+1.  Maintainers and Contributors MUST have a Platform account and SHOULD use their real names or a well-known alias.
+2.  A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.
+3.  A patch MUST adhere to the code style guidelines of the project if these are defined.
+4.  A patch MUST adhere to the “Evolution of Public Contracts” guidelines defined below.
+5.  A patch SHALL NOT include non-trivial code from other projects unless the Contributor is the original author of that code.
+6.  A patch MUST compile cleanly and pass project self-tests on at least the principal target platform.
+7.  A patch commit message MUST consist of a single short (less than 50 characters) line stating the problem (“Problem: …") being solved, followed by a blank line and then the proposed solution (“Solution: …").
+8.  A “Correct Patch” is one that satisfies the above requirements.
+
+### 2.4. Development Process
+
+1. Change on the project SHALL be governed by the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
+2. To request changes, a user SHOULD log an issue on the project Platform issue tracker.
+3. The user or Contributor SHOULD write the issue by describing the problem they face or observe.
+4. The user or Contributor SHOULD seek consensus on the accuracy of their observation, and the value of solving the problem.
+5. Users SHALL NOT log feature requests, ideas, suggestions, or any solutions to problems that are not explicitly documented and provable.
+6. Thus, the release history of the project SHALL be a list of meaningful issues logged and solved.
+7. To work on an issue, a Contributor SHALL fork the project repository and then work on their forked repository.
+8. To submit a patch, a Contributor SHALL create a Platform pull request back to the project.
+9. A Contributor SHALL NOT commit changes directly to the project.
+10. If the Platform implements pull requests as issues, a Contributor MAY directly send a pull request without logging a separate issue.
+11. To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+12. To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+13. Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
```

Comment:
I think there's value in putting a hard deadline to respond here, to remove ambiguity and the "Oh I thought I waited long enough" style of conflict

Reply:
Updated.